Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -8729,7 +8736,10 @@ void WebPageProxy::createNewPage(IPC::Connection& connection, WindowFeatures&& w
             m_browsingContextGroup.copyRef(),
             originatingFrameInfoData.frameID
         } });
-        configuration->setOpenedSite(openerFrame->frameProcess().site());
+        auto site = WebCore::Site(openerFrame->url());
```

Comment:
nit: auto doesn't make anything better on this line.  May as well just specify the type, and you can reduce the number of keywords!

Reply:
oh yeah, I'd use `WebCore::Site` on the LHS.